app/controllers/v1/released_controller.rb

```diff
@@ -7,7 +7,7 @@ class ReleasedController < ApplicationController
     # Query for all druids that have a certain release tag.
     # This is used by the sitemap generator in PURL.
     def show
-      release_tag = params[:id]
+      release_tag = params[:druid]
```

Comment:
This is a release tag, not a druid.

Reply:
Thanks, I didn't catch that at all. Isn't it pretty weird to have `GET /released/*` and `PUT /released/*` do totally different things?

Reply:
^^ cc @jcoyne